contracts/src/BlobstreamX.sol

```diff
@@ -24,10 +24,10 @@ contract BlobstreamX is
 
     /// @notice The maximum number of blocks that can be skipped in a single request.
     /// Source: https://github.com/celestiaorg/celestia-core/blob/main/pkg/consts/consts.go#L43-L44
-    uint64 public DATA_COMMITMENT_MAX = 1000;
+    uint64 public constant DATA_COMMITMENT_MAX = 1000;
 
     /// @notice Nonce for proof events. Must be incremented sequentially.
-    uint256 public state_proofNonce = 1;
+    uint256 public state_proofNonce;
```

Comment:
nit: is there a reason this isn't stateProofNonce or proofNonce?

Reply:
This matches the current variable naming in `Blobstream`. Can definitely change later on, but for now will leave as this.